src/doom/g_game.c

```diff
@@ -1319,6 +1319,15 @@ int cpars[32] =
     240,150,180,150,150,300,330,420,300,180,	// 21-30
     120,30					// 31-32
 };
+
+// Chex Quest Par Times
+int qpars[4][10] = 
+{ 
+    {0}, 
```

Comment:
This should list all ten zeroes or GCC might warn about it being incomplete.

Reply:
It won't. If you initialize one element of an array to whatever value, all the following ones are implicitly initialized to 0.